machine_learning/management/commands/ml_smoothing.py

```diff
@@ -2,13 +2,15 @@
 
 import numpy as np
 import tensorflow as tf
+
 from django.core.management.base import BaseCommand
 
+from Key import Key
+from plot import *
+
+from cycling.models import *
```

Comment:
I am trying to avoid circular dependencies between the different subfolders (because django uses dependencies to create a database migration).
please import only what is needed